### ISSUE TITLE
rpi_ws281x dependency needs updating

### ISSUE THREAD

**Issue by penfold42** [tvdzwan/hyperion#546]
How do we change the dependent version?
 	rpi_ws281x @ 34c917e fails for some RPI3 devices because the model detection fails.

master has the changes that I created to add an additional model number.

This is the error you get:
Application build time: Mar 12 2016 15:15:29
QCoreApplication initialised
Selected configuration file: /etc/hyperion.config.json.pwm
ColorTransform 'default' => [0; 81]
Device configuration:
{
        "colorOrder" : "grb",
        "name" : "MyPi",
        "type" : "ws281x"
}

Unable to initialize ws281x library.
terminate called after throwing an instance of 'int'
Aborted

**Comment by brindosch** [comment]
We have no new images, the rpi image is just replaced with a working on for ws2812. on RPi1.
The latest commit from redPanther and yours through the lib is not implemented!
We hope to start soon with a better image releasing system :)

**Comment by penfold42** [comment]
Huh ? I'm confused. I just pulled down the tar file in the deploy folder and it seems to have the new ws281x driver in it

**Comment by brindosch** [comment]
Now i am confused. your commit last night at the lib is not in the bin. check the build date above.
Is this subproject commit change necessary or thought you it won´t work with it?

**Comment by penfold42** [comment]
Note - I have no real idea what in doing with git so if something looks stupid it could very well be!

The sub module we use - rpi_281x only worked on some rpi3s, not all of them because it's pi version detection was missing another check.

I fixed it in its repository and my pull request was merged a couple of days ago.

What I'm not sure of is how in hyperion a git repository to correctly update the sub module reference to point to an updated version that includes my change.
